.changeset/blue-items-repeat.md

```diff
@@ -0,0 +1,8 @@
+---
+"@primer/components": patch
+---
+
+Hide divider before `ActionList.Group`s with filled header
+Align `Item` description to when rendered in-line
+Add `selectionVariant: 'multiple'` for `Item`s.  These will use a checkbox input instead of a checkmark icon for selected state
+Allow `focusZoneSettings` to be passed into `AnchoredOverlay`
```

Comment:
If it's not too much trouble, it might be nice to put all these distinct changes in their own changeset.